src/session.ts

```diff
@@ -91,7 +91,7 @@ export default function session(options: Options = {}) {
         [isNew]: true,
         cookie: {
           path: cookieOpts.path || "/",
-          httpOnly: cookieOpts.httpOnly || true,
+          httpOnly: cookieOpts.httpOnly ?? true,
```

Comment:
Thanks. Could you instead do a `typeof == "boolean"` check instead of `??` since we want to maintain compatibility with older Node version?